generator/schemas/zz_generated.markerhelp.go

```diff
@@ -0,0 +1,42 @@
+// +build !ignore_autogenerated
+
+// Generated for the devfile generator
+
+// Code generated by helpgen. DO NOT EDIT.
+
+package schemas
+
+import (
+	"sigs.k8s.io/controller-tools/pkg/markers"
+)
+
+func (GenerateJSONSchema) Help() *markers.DefinitionHelp {
```

Comment:
I see that's generated `go run sigs.k8s.io/controller-tools/cmd/helpgen generate:headerFile=../header.go.txt,year=2020 paths=.` but it's not clear at least for me - when it's supposed to happen? Triggered manually after the main file is edited?
If it's not there yet - should we include it into build process to make sure it has actual state on each PR?

Reply:
I just missed it. Thanks for pointing this out.

This was fixed in commit https://github.com/devfile/api/pull/133/commits/66c0dd08da2c596e5c77995eefd59a9579975fc0